### DISCUSSION TITLE
New built

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#42] [General]
Hi I just saw the new built under actions with the note: Use r8152 oot drivers.
What exactly has been changed here? And are all changes from the original built in here as well? 
I mean the changes from the 'official' releases mentioned here:
https://github.com/nicholas-opensource/OpenWrt-Autobuild/releases

**Comment by nicholas-opensource** [comment]
It brought bad results, revert
